docs/src/examples/components/TextArea/Types/TextAreaInvertedExample.shorthand.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react'
+import { Grid, TextArea } from '@stardust-ui/react'
+
+const TextAreaExample = () => (
+  <div>
+    <Grid
+      styles={({ theme: { siteVariables } }) => ({
+        backgroundColor: siteVariables.colorScheme.default.background2,
+        padding: '20px',
+      })}
+    >
+      <TextArea inverted placeholder="Inverted color text area..." />
+    </Grid>
+    <Grid
+      styles={({ theme: { siteVariables } }) => ({
+        backgroundColor: siteVariables.colorScheme.default.background,
+        padding: '20px',
+      })}
+    >
+      <TextArea placeholder="Default text area..." />
+    </Grid>
+  </div>
+)
+
+export default TextAreaExample
```

Comment:
```suggestion
export default TextAreaExampleInverted
```

Please also change filename